### ISSUE TITLE
2 characters jump?

### ISSUE THREAD

**Issue by mihaigalos** [fcsonline/tmux-thumbs#150]
I really like your plugin and was wondering if I could get it to jump anywhere on the screen based on 2 characters instead of one, similar to https://github.com/roy2220/easyjump.tmux.

leader+space -> input dialog for 2 letters -> jump there.

Any thoughts?

**Comment by fcsonline** [comment]
Mmmmm... this is exactly the current behaviour. I mean, if there is a short list of matches, then appears a single character hint, but if the list is longer, two characters hints appear.

**Comment by mihaigalos** [comment]
Precisely. But what I'm asking is the ability to jump anywhere (not copy). Currently, leader+space activates `tmux-thumbs` and directly annotates the 2 letters.
What I'm asking is the ability to:
* leader+space
* show a dialog for 2 chars to search for (anywhere in the screen)
* type 2 chars -> highlight and annotate matches
* type the  char(s) to jump there
